src/ds/logger.h

```diff
@@ -134,13 +171,72 @@ namespace logger
   {
     bool operator==(LogLine& line)
     {
-      std::cout << line.ss.str() << std::flush;
+      write(line.ss.str());
 
       if (line.log_level == Level::FATAL)
         throw std::logic_error("Fatal: " + line.ss.str());
 
       return true;
     }
+
+    static void write(const std::string& s)
+    {
+      // When logging from host code, print local time.
+      ::timespec ts;
+      ::timespec_get(&ts, TIME_UTC);
+      std::tm now;
+      ::localtime_r(&ts.tv_sec, &now);
+
+      std::cout << fmt::format(
+                     "{:%Y-%m-%d %H:%M:%S}.{:0<6}        ",
```

Comment:
Can you add a comment with an example of the output from this? Easier to understand the fixed width spacing from an example.

Reply:
Will do